TeamProject/GlobalWarmingGame/SoundFactory.cs

```diff
@@ -10,65 +10,49 @@ namespace GlobalWarmingGame
-            songs.Add("enemy_zone", content.Load<Song>("sound/songs/enemy_zone"));
-            sounds.Add("wood_chop", content.Load<SoundEffect>(@"sound/sounds/wood_chop"));
-            sounds.Add("rabbit_death", content.Load<SoundEffect>(@"sound/sounds/rabbit_death"));
-            sounds.Add("stone_pickup", content.Load<SoundEffect>(@"sound/sounds/stone_pickup"));
-        }
-        static public void PlayGameSoundtrack()
-        {
-            song = songs["main"];
-            MediaPlayer.IsRepeating = true;
-            MediaPlayer.Play(song);
-            MediaPlayer.MediaStateChanged += MediaPlayer_MediaStateChanged;
+            songs = new Dictionary<PFSong, Song>
+            {
+                { PFSong.Menu,      content.Load<Song>(@"sound/songs/menu") },
+                { PFSong.Game,      content.Load<Song>(@"sound/songs/ColdAtmosphericMusic") },
+                { PFSong.EnemyZone, content.Load<Song>(@"sound/songs/enemy_zone") }
+            };
+            sounds = new Dictionary<Sound, SoundEffect>
+            {
+                { Sound.WoodChop,       content.Load<SoundEffect>(@"sound/sounds/wood_chop") },
+                { Sound.RabbitDeath,    content.Load<SoundEffect>(@"sound/sounds/rabbit_death") },
+                { Sound.StonePickup,    content.Load<SoundEffect>(@"sound/sounds/stone_pickup") }
+            };
         }
-        static public void PlayGameMenuSong()
+        static public void PlaySong(PFSong s, bool repeate = true)
         {
-            song = songs["menu"];
-            MediaPlayer.IsRepeating = true;
+            song = songs[s];
+            MediaPlayer.IsRepeating = repeate;
             MediaPlayer.Play(song);
             MediaPlayer.MediaStateChanged += MediaPlayer_MediaStateChanged;
         }
-        public static void MediaPlayer_MediaStateChanged(object sender, System.EventArgs e)
+        private static void MediaPlayer_MediaStateChanged(object sender, System.EventArgs e)
         {
             MediaPlayer.Volume -= 100f;
             MediaPlayer.Play(song);
         }
-        public static void PlaySoundEffect(Sound sound)
-        {
-            SoundEffect mySound = null;
-            switch (sound)
-            {
-                case Sound.wood_chop:
-                    mySound = sounds["wood_chop"];
-                    break;
-                case Sound.rabbit_death:
-                    mySound = sounds["rabbit_death"];
-                    break;
-                case Sound.stone_pickup:
-                    mySound = sounds["stone_pickup"];
-                    break;
-                default:
-                    throw new NotImplementedException(sound + " has not been implemented");
-            }
-             
-            mySound.Play();
-        } 
+        public static void PlaySoundEffect(Sound sound) => sounds[sound].Play();
     }
 }
 public enum Sound
 {
-    wood_chop,
-    rabbit_death,
-    stone_pickup
+    WoodChop,
```

Comment:
C# Enum naming scheme asks for PascalCase identifiers
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum